Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -271,6 +270,18 @@ extension _ChatChannel {
                 .map { $0.asModel() }
         }
         
+        let fetchWatchers: () -> [_ChatUser<ExtraData.User>] = {
+            UserDTO
+                .loadLastActiveWatchers(for: cid, context: context)
+                .map { $0.asModel() }
+        }
+        
+        let fetchMembers: () -> [_ChatChannelMember<ExtraData.User>] = {
+            MemberDTO
+                .loadLastActiveMembers(in: cid, context: context)
+                .map { $0.asModel() }
+        }
+        
```

Comment:
We should capture the behavior of those two variables in tests - does it return the correct entities in the correct order with the correct limit.